### ISSUE TITLE
[Bug]: Header items are for the whole instance vs. just the current database

### ISSUE THREAD

**Issue by 99littlebugs** [RedisInsight/RedisInsight#375]
**Preconditions** version 2.0

**Steps to reproduce** Create keys in more than one redis database

**Actual behavior** Observe that the header shows the number of keys, connections, and others across the whole instance

**Expected behavior** Either to show the stats (such as number of keys) for just this database, or to make it clearer that those stats are on a per instance level. Considering that each database index needs its own connection profile (and there is no quick switcher), it was unexpected that anything on the UI was for the entire instance. 

**Screenshots**
![image](https://user-images.githubusercontent.com/7087640/158278701-c0945525-ea7f-4aaa-bb0d-dd528e7153d3.png)

**Additional context** MacOS, RedisInsight 2.0

**Comment by ViktarStarastsenka** [comment]
Hi @99littlebugs , the overview displays information only about the current database.
The reason for your case is that the Overview parameters are updated automatically, while the list of keys in Browser should be updated manually.
You can refresh the Browser content ("refresh" icon to the right of filtering) to see new keys.
Please let me know if it worked.

**Comment by 99littlebugs** [comment]
Nope. The database index 2 (top left) has no keys in it. Other databases though on the same redis instance did have data - which is what was being shown on the top right.

**Comment by ViktarStarastsenka** [comment]
Got it, so you use different database indexes.
Thanks for raising this issue, I'll see what we can do here.

**Comment by ViktarStarastsenka** [comment]
@99littlebugs , will it work if the tooltip displays total number of keys and the number of keys in this logical database?
![image](https://user-images.githubusercontent.com/99594890/165084613-83d587c8-6dd9-406e-bc7d-7a4763894b3d.png)

**Comment by 99littlebugs** [comment]
Yes, I think that would work well.

**Comment by ViktarStarastsenka** [comment]
@99littlebugs , fixed with 2.4.0 (released today).
